packages/opds-web-client/src/components/Book.tsx

```diff
@@ -83,6 +93,11 @@ export default class Book<P extends BookProps> extends React.Component<P, void>
               { this.fields().map(field =>
                 field.value ? <div className={field.name.toLowerCase().replace(" ", "-")} key={field.name}>{field.name}: {field.value}</div> : null
               ) }
+              {
+                bookMedium && (
+                  <span>{this.getMediumSVG(bookMedium, false)}</span>
```

Comment:
I think this might look better with the label? And I'd probably put it above the publisher.

Reply:
Updated.